src/SDKs/Authorization/Authorization.Tests/Tests/BasicTests.cs

```diff
@@ -729,15 +733,26 @@ public void RoleDefinitionCreateTests()
 
                 RoleDefinition createOrUpdateParams;
                 var roleDefinitionId = GetValueFromTestContext(Guid.NewGuid, Guid.Parse, "RoleDefinition1");
-                string currentSubscriptionId = "/subscriptions/" + client.SubscriptionId;
-                string fullRoleId = currentSubscriptionId + RoleDefIdPrefix + roleDefinitionId;
+                string currentSubscriptionId = "/subscriptions/" + client.SubscriptionId + "/" + ResourceGroup;
+                string fullRoleId = "/subscriptions/" + client.SubscriptionId + RoleDefIdPrefix + roleDefinitionId;
 
                 Guid newRoleId = GetValueFromTestContext(Guid.NewGuid, Guid.Parse, "RoleDefinition2"); 
-                var resourceGroup = "newtestrg";   
-                string resourceGroupScope = currentSubscriptionId + "/resourceGroups/" + resourceGroup;
-
-                // Create a custom role definition
-                try
+                string resourceGroupScope = currentSubscriptionId;
+				
+				// create resource group,This works only if logged in using Username/Password method
+				var resourceClient = PermissionsTests.GetResourceManagementClient(context);
+				try
+				{
+					resourceClient.ResourceGroups.CreateOrUpdate(
+						"AzureAuthzSDK1",
+						new ResourceGroup
+						{ Location = "westus" });
```

Comment:
Avoid hard-coding locations - location should be decided based on the available locations for the necessary providers.  It is OK to prefer a location however.  Feel free to file an issue to fix this in the next sprint if you can't fix right away.